api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -451,6 +451,17 @@ default void playSound(final @NonNull Sound sound) {
   default void playSound(final @NonNull Sound sound, final double x, final double y, final double z) {
   }
 
+  /**
+   * Plays a sound from an emitter, usually an entity.
```

Comment:
I'd like to see this clarify what it means to play a sound "from" an entity -- it means that the location of the sound will follow the location of that entity, right?